server/build_without_gradle.sh

```diff
@@ -9,14 +9,24 @@
 #
 #     BUILD_DIR=my_build_dir ./build_without_gradle.sh
 
-set -e
+set -eo pipefail
```

Comment:
Isn't it too violent?

```bash
set -eo pipefail
result=$(echo a | grep b)
```

Reply:
It depends on what behavior you want here. You might want something like this:`echo a | grep b || true`, instead.